test/sql/storage/compression/roaring/roaring_compression_ratio.test_slow

```diff
@@ -0,0 +1,211 @@
+# name: test/sql/storage/compression/roaring/roaring_compression_ratio.test_slow
+# description: Assert roaring compression ratio is within reasonable margins for each container type
+# group: [roaring]
+
+require block_size 262144
+
+load __TEST_DIR__/test_roaring.db
+
+statement ok
+set variable dataset_size = 120_000_000;
+
+#### Array container Roaring Compression ratio calculation:
+# For single row group
+# 60 vectors with 82 non-null values per vector
+# Total compressed bytes = (60 * 2 (metadata)) + (60 * (82 * 2)) = 9960 bytes
```

Comment:
These calculations need to be updated to account for the compressed array containers